page.go

```diff
@@ -86,6 +87,8 @@ type PageCreateRequest struct {
 	Children   []Block    `json:"children,omitempty"`
 }
 
+type Url string
```

Comment:
It seems like we don't need the type alias here. Let's parse the `url` as a string

Reply:
Changed `URL` property type to `string`